ambry-api/src/main/java/com/github/ambry/config/ServerConfig.java

```diff
@@ -92,14 +92,21 @@
   @Config("server.validate.request.based.on.store.state")
   @Default("false")
   public final boolean serverValidateRequestBasedOnStoreState;
-  
+
   /**
    * True to enable ambry server handling undelete requests.
    */
   @Config("server.handle.undelete.request.enabled")
   @Default("false")
   public final boolean serverHandleUndeleteRequestEnabled;
 
+  /**
+   * Implementation class for HelixAccountServiceFactory
+   */
+  @Config("server.account.service.factory")
+  @Default("com.github.ambry.account.InMemAccountServiceFactory")
```

Comment:
Why not use the same factory as RestServer does? 
`com.github.ambry.account.InMemoryUnknownAccountServiceFactory`

Reply:
You are right. InMemAccountServiceFactory is for testing. I've updated.